pkg/scheduler/uthelper/helper.go

```diff
@@ -180,6 +185,14 @@ func (test *TestCommonStruct) CheckBind(caseIndex int) error {
 		}
 	}
 
+	if test.MinimalBindCheck {
```

Comment:
What's the reason add a new field MinimalBindCheck here and move the code:
```go
if test.ExpectBindsNum != len(test.ExpectBindMap) {
		return fmt.Errorf("invalid setting for binding check: want bind count %d, want bind result length %d", test.ExpectBindsNum, len(test.ExpectBindMap))
	}
```
to the back?

Reply:
Because sometimes we just want check the binding num and no need to check ExpectBindMap, because we may not know the exact bind result(pod->node), and also this value is false by default, so it will not break the existing UTs.

Reply:
This return should put after the next select case, in case that if ExpectBindsNum is 0, but actually there are binds.